src/Common/src/System/Security/Cryptography/DerSequenceReader.cs

```diff
@@ -110,6 +110,67 @@ internal byte[] ReadNextEncodedValue()
             return encodedValue;
         }
 
+        internal byte[] ReadRemainingData()
+        {
+            int remainingBytes = _end - _position;
```

Comment:
What guarantees that remainingBytes is positive here?  Is it possible it's negative?  Is it commonly going to be 0?